saleor/graphql/checkout/tests/mutations/test_checkout_complete.py

```diff
@@ -289,6 +299,128 @@ def test_checkout_complete(
     order_confirmed_mock.assert_called_once_with(order)
 
 
```

Comment:
Could you add a test for situations when we have metadata stored in checkout and we perform `CheckoutComplete` with additional metadata(But updating any key stored in the original checkout object)? 

example date:
Checkout_meta - {"a": 1, "b":2}
Meta passed to mutation -  {"a":4, "c":3}
Result: {"a":4, "b":2, "c":3}